src/Symfony/Component/Notifier/Bridge/FreeMobile/FreeMobileTransport.php

```diff
@@ -75,5 +76,9 @@ protected function doSend(MessageInterface $message): void
 
             throw new TransportException(sprintf('Unable to send the SMS: error %d: ', $response->getStatusCode()).($errors[$response->getStatusCode()] ?? ''), $response);
         }
+
+        $message = new SentMessage($message, uniqid());
```

Comment:
I would allow the id to be null instead of generating a random one.

Reply:
I removed it from __construct then.